tests/test_multilabel_classification.py

```diff
@@ -718,3 +722,71 @@ def test_invalid_alpha(self):
         for alpha in [-0.5, 1.5]:
             with pytest.raises(ValueError, match=partial_error_msg):
                 ml_scorer.exponential_moving_average(np.ones(5).reshape(1, -1), alpha=alpha)
+
+
+def flip_labels(label, flip_prob):
+    return 1 - np.array(label) if np.random.rand() < flip_prob else label
+
+
+@composite
+def cleanlab_data_strategy(draw):
+    num_classes = draw(st.integers(min_value=2, max_value=3))
+    num_samples = draw(st.integers(min_value=10, max_value=50))
+
+    # Generate true labels as one-hot encoded vectors for multi-label
+    true_labels = draw(
+        st.lists(
+            st.lists(
+                st.integers(min_value=0, max_value=1),
+                min_size=num_classes,
+                max_size=num_classes,
+            ),
+            min_size=num_samples,
+            max_size=num_samples,
+        )
+    )
+
+    # Generate noise matrix for multi-label
+    flip_prob = 0.2
+    noisy_labels = np.array([flip_labels(label, flip_prob) for label in true_labels])
+
+    # Modify the noisy_labels using your logic
+    for i in range(num_samples):
+        for j in range(num_classes):
+            if draw(st.floats(min_value=0, max_value=1)) < 0.2:
+                noisy_labels[i][j] = 0
+
+    # Generate predicted probabilities for each class for each sample
+    pred_probs = draw(
+        st.lists(
+            st.lists(
+                st.floats(min_value=0, max_value=1),
+                min_size=num_classes,
+                max_size=num_classes,
+            ),
+            min_size=num_samples,
+            max_size=num_samples,
+        )
+    )
+
+    for i in range(num_samples):
+        for j in range(num_classes):
+            if draw(st.floats(min_value=0, max_value=1)) < 0.1:
+                # Set some probability values to exactly 0.5
+                pred_probs[i][j] = 0.5
+    return true_labels, noisy_labels, np.array(pred_probs)
+
+
+class TestMultiLabel:
+    @given(cleanlab_data_strategy())
+    @settings(deadline=10000)
+    def test_find_label_issues(self, data):
+        true_labels, noisy_labels, pred_probs = data
+        noisy_labels_list = onehot2int(noisy_labels)
+        is_issue = filter.find_label_issues(
+            labels=noisy_labels_list, pred_probs=np.array(pred_probs)
+        )
+        threshold = 0.5
+        pred_labels = (pred_probs >= threshold).astype(int)
+        equal_pred = np.where(np.all(pred_labels == noisy_labels, axis=1), True, False)
```

Comment:
The `np.where(..., True, False)` is redundant.

```suggestion
        equal_pred = np.all(pred_labels == noisy_labels, axis=1)
```